optimum/habana/transformers/training_args.py

```diff
@@ -152,6 +152,11 @@ class GaudiTrainingArguments(TrainingArguments):
         },
     )
 
+    use_fused_gelu: Optional[bool] = field(
+        default=False,
```

Comment:
Shouldn't we use it by default? Or is there any use case where it should not be enabled?

Reply:
yes, I will set the default value to True